src/ui-kit/components/modal/modal.template.html

```diff
@@ -1,6 +1,6 @@
 <div *ngIf="show" class="modal-backdrop"></div>
 <div aria-live="assertive" aria-modal="true" #modalRoot *ngIf="show" [attr.id]="id" role="dialog" class="modal-dialog"
-	(keydown.esc)="closeOnEscape ? closeModal() : 0" (click)="closeOnOutsideClick ? closeModal() : 0" aria-labelledby="dialog-title" aria-describedby="dialog-description">
+	(keydown.esc)="closeOnEscape ? closeModal() : 0" (clickOutside)="closeOnOutsideClick ? closeModal() : 0" aria-labelledby="dialog-title" aria-describedby="dialog-description">
```

Comment:
Is there a reason you are not adding this
sam-click-outside as an attribute

Reply:
added, i had configured the implementation with closeOnOutsideClick=false and thought that was the entire fix

re-testing